experiments/template_SLTC.py

```diff
@@ -220,6 +235,7 @@ def main():
         trainer.log_metrics("train", metrics)
         trainer.save_metrics("train", metrics)
         trainer.save_state()
+        wandb.log(metrics)
```

Comment:
Is this step really necessary? Doesn't the logging happen with trainer.log_metrics?